### DISCUSSION TITLE
Quick way to get vehicle location

### DISCUSSION THREAD

**Discussion by kp-bit** [pypolestar/polestar_api#123] [Show and tell]
Create a Nabu Casa account and install the Home Assistant app in Android Automotive. Connect it to the Nabu Casa URL and allow it to share location at all times. That will give you a device_tracker.polestar (or whatever name you choose) in Home Assistant.

**Comment by leeyuentuen** [comment]
without nabu casa account it is also possible, just use the home assistant app in the car. but this is not the way that we want from the polestar api. but it is a workaround

**Reply by kp-bit** [reply]
Yeah would be more elegant to pull it through the API, but an acceptable workaround, at least in my case...
